AnkiDroid/src/main/java/com/ichi2/anki/MyAccount.java

```diff
@@ -176,6 +184,24 @@ private void initAllContentViews() {
         mUsername = mLoginToMyAccountView.findViewById(R.id.username);
         mPassword = mLoginToMyAccountView.findViewById(R.id.password);
         mPasswordLayout = mLoginToMyAccountView.findViewById(R.id.password_layout);
+        mAnkidroid_logo = mLoginToMyAccountView.findViewById(R.id.ankidroid_logo);
+
+        //checking if device is in horizontal mode or not .
+        myOrientationEventListener = new OrientationEventListener(this,SensorManager.SENSOR_DELAY_NORMAL)
+        {
+            @Override
+            public void onOrientationChanged(int orientation)
+            {
+                PORTRAIT_MODE = ((orientation < 100) || (orientation > 280));
+                //if device is in horizontal mode then screen might not have enough space for ankidroid logo
+                //so we will invisible logo for horizontal mode only
+                if(PORTRAIT_MODE)
```

Comment:
Style: [one space before an if](https://github.com/ankidroid/Anki-Android/wiki/Code-style#white-space-should-be-used-in-the-following-cases)

```suggestion
                if (PORTRAIT_MODE)
```